From cc21b228f192d944b5676e948b16e64e446bc698 Mon Sep 17 00:00:00 2001
From: Nikhil Badola <nikhil.badola@freescale.com>
Date: Wed, 29 Oct 2014 08:55:54 +0530
Subject: [PATCH 088/124] drivers: usb: fsl: Change string format for errata
 property

Change string format for errata property in USB node that is
checked in device tree for applicablilty of corresponding erratum

Signed-off-by: Nikhil Badola <nikhil.badola@freescale.com>
Change-Id: Ic07fa5b5a8ce93ac3845332f23b3b71338956266
Reviewed-on: http://git.am.freescale.net:8181/22463
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Ramneek Mehresh <ramneek.mehresh@freescale.com>
Reviewed-by: Matthew Weigel <Matthew.Weigel@freescale.com>
---
 drivers/usb/host/fsl-mph-dr-of.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/usb/host/fsl-mph-dr-of.c b/drivers/usb/host/fsl-mph-dr-of.c
index 49a23ad..68cf686 100644
--- a/drivers/usb/host/fsl-mph-dr-of.c
+++ b/drivers/usb/host/fsl-mph-dr-of.c
@@ -218,30 +218,30 @@ static int fsl_usb2_mph_dr_of_probe(struct platform_device *ofdev)
 	pdata->controller_ver = usb_get_ver_info(np);
 
 	/* Activate workaround for USB erratum-A00XXXX if
-	 * fsl,erratum-a00XXXX property is defined for
+	 * fsl,usb-erratum-a00XXXX property is defined for
 	 * affected socs
 	 */
-	if (of_get_property(np, "fsl,usb_erratum-a005275", NULL))
+	if (of_get_property(np, "fsl,usb-erratum-a005275", NULL))
 		pdata->has_fsl_erratum_a005275 = 1;
 	else
 		pdata->has_fsl_erratum_a005275 = 0;
 
-	if (of_get_property(np, "fsl,usb_erratum-a005697", NULL))
+	if (of_get_property(np, "fsl,usb-erratum-a005697", NULL))
 		pdata->has_fsl_erratum_a005697 = 1;
 	else
 		pdata->has_fsl_erratum_a005697 = 0;
 
-	if (of_get_property(np, "fsl,usb_erratum-a007792", NULL))
+	if (of_get_property(np, "fsl,usb-erratum-a007792", NULL))
 		pdata->has_fsl_erratum_a007792 = 1;
 	else
 		pdata->has_fsl_erratum_a007792 = 0;
 
-	if (of_get_property(np, "fsl,usb_erratum_a006918", NULL))
+	if (of_get_property(np, "fsl,usb-erratum-a006918", NULL))
 		pdata->has_fsl_erratum_a006918 = 1;
 	else
 		pdata->has_fsl_erratum_a006918 = 0;
 
-	if (of_get_property(np, "fsl,usb_erratum_14", NULL))
+	if (of_get_property(np, "fsl,usb-erratum-14", NULL))
 		pdata->has_fsl_erratum_14 = 1;
 	else
 		pdata->has_fsl_erratum_14 = 0;
-- 
2.1.0.27.g96db324

